sources/packages/backend/libs/services/src/sfas/sfas-application.service.ts

```diff
@@ -73,6 +75,48 @@ export class SFASApplicationService extends DataModelService<SFASApplication> {
     return +(total?.sum ?? 0);
   }
 
+  /**
+   * Fetch the SFAS full time application for the student and the latest
+   * application which has the end date within 730 days.
+   * @param studentId student id.
+   * @returns SFASSignedMSFAA which contains the SFAS Signed MSFAA
+   * and latest application end date.
+   */
+  async getIndividualFullTimeApplicationByIndividualId(
```

Comment:
The method name and comment seem misleading since the method is returning a valid MSFAA.
I would suggest renaming it to `getValidMSFAA`.